### DISCUSSION TITLE
Invoking edge functions with abort signal

### DISCUSSION THREAD

**Discussion by jxw125** [supabase/supabase#27660] [Questions]
Hello!
The documentation shows a way of invoking a DB call with an abortSignal to cancel the request https://supabase.com/docs/reference/javascript/db-abortsignal, but I couldn't find anything similar for invoking an edge function? Is such a feature implemented?

**Comment by GaryAustin1** [comment]
No.  But it has been requested before.
https://github.com/orgs/supabase/discussions/17715

**Comment by dsc8x** [comment]
Here we go https://github.com/supabase/functions-js/pull/93
